tsl/test/sql/partialize_finalize.sql

```diff
@@ -28,6 +31,8 @@ insert into t1 select * from t1;
 select a, _timescaledb_internal.finalize_agg( 'count("any")', null, null, null, partial, cast('1' as int8) ) from t1
 group by a order by a ;
 
+SET enable_partitionwise_aggregate = off;
```

Comment:
Moving this to line 68 will cover the NUMERIC case with internal type as well.

Reply:
Moved.